src/components/MainNavBar/MainNavBar.tsx

```diff
@@ -39,66 +36,100 @@ const MainNavBar = () => {
 
   const { authState } = useOktaAuth();
   return (
-    <Nav>
-      <InnerNav>
-        <InnerMost>
-          <Logo to="/measures">
-            <img src={logo} alt="MADiE Logo" />
-          </Logo>
-        </InnerMost>
+    <nav>
+      <div className="inner">
+        <div>
+          <NavLink to="/measures">
+            <img src={logo} alt="MADiE Logo" id="logo" />
+          </NavLink>
+          <div className="divider" />
+          <h2 className="header-info">
+            Measure Authoring Development <br /> Integrated Development
```

Comment:
MADiE full form needs to be corrected(Please check with Lindsay/Stan whether it is `Measure Authoring Development Integrated Environment` or `Measure Authoring Development & Integrated Environment`)
```suggestion
            Measure Authoring Development <br /> Integrated Environment
```

Reply:
also, it is outside of `<NavLink>`. so, not sure it is clickable to home like logo